android/src/main/java/com/onesignal/rnonesignalandroid/RNOneSignal.java

```diff
@@ -172,7 +172,7 @@ private void removeObservers() {
 
     private void removeHandlers() {
         if(!oneSignalInitDone) {
-            Log.i("OneSignal", "OneSignal React-Native SDK not initialized yet. Could not remove handlers.");
+            Logging.debug("OneSignal React-Native SDK not initialized yet. Could not remove handlers.", null);
```

Comment:
nit, no need to pass in `null` as the 2nd parameter to our OneSignal `Logging` functions, it as optional parameter.

Reply:
Since we are calling this Kotlin-defined method from Java, we have to pass the 2nd parameter.